docs/source/traits_api_reference/trait_types.rst

```diff
@@ -147,6 +147,8 @@ Traits
 
 .. autoclass:: Type
 
+.. autoclass:: Subclass
```

Comment:
It looks as though Sphinx is clever enough to automatically say "alias of traits.trait_types.Type" here. Nice. I was expecting to see the class details duplicated.